.eslintrc.js

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
indent this file with 2 spaces instead of 4

Reply:
Done